### PR TITLE
Add scrollbar mixin

### PR DESCRIPTION
**Pull request by Faradey27**
WIP till thinking how to expose for external users

### CODE REVIEW

File: stories/ScrollBar/Readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+How to use scroll bar mixin in your project
```
Comment (argshook) [inline stories/ScrollBar/Readme.md]: if it's for internal use, i don't think we need to  show it in storybook

File: stories/ScrollBar/ExampleStandard.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+class ExampleStandard extends Component {
+  static propTypes = {
+    onChange: PropTypes.func,
+    theme: PropTypes.string
+  };
+
+  render() {
+    return (
+      <div>
+        <MessageBoxFunctionalLayout
```
Comment (shlomitc) [inline stories/ScrollBar/ExampleStandard.js]: shouldn't this be a built-in functionality of the `MessageBox` component anyway?

File: stories/ScrollBar/Readme.md (hunk 1 above)
Comment (Faradey27) [reply stories/ScrollBar/Readme.md]: no, it for internal and external

File: stories/ScrollBar/ExampleStandard.js (hunk 2 above)
Comment (Faradey27) [reply stories/ScrollBar/ExampleStandard.js]: i made it default in MessageBox in this PR and used MessageBox as example,
in general i can add any other example(just divs with heighs)

File: stories/ScrollBar/Readme.md (hunk 1 above)
Comment (Faradey27) [reply stories/ScrollBar/Readme.md]: so i added an example how our clients can use our mixins and add new way to export scss files
@argshook

File: stories/ScrollBar/ExampleStandard.js (hunk 2 above)
Comment (shlomitc) [reply stories/ScrollBar/ExampleStandard.js]: Yes, so I prefer some new example that shows how to implement it.
Also - the code snippet should appear

File: src/scrollBarsMixins.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  &::-webkit-scrollbar-button {
+    display: none;
+    height: 6px;
+  }
+  &::-webkit-scrollbar-thumb {
+    /* https://stackoverflow.com/a/16829330/5343330 */
+    background-color: $B30;
+    background-clip: content-box;
+    border-radius: 4px * 2; // we need to double border-radius to receive 4px
+    border: 4px solid transparent; // for width emulation
```
Comment (milkyfruit) [inline src/scrollBarsMixins.scss]: please increase the border to 5px:
image

File: src/scrollBarsMixins.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  }
+  &::-webkit-scrollbar-button {
+    display: none;
+    height: 6px;
+  }
+  &::-webkit-scrollbar-thumb {
+    /* https://stackoverflow.com/a/16829330/5343330 */
+    background-color: $B30;
+    background-clip: content-box;
+    border-radius: 4px * 2; // we need to double border-radius to receive 4px
```
Comment (milkyfruit) [inline src/scrollBarsMixins.scss]: 8px border radius still looks not right in a chrome browser. Increasing it to 10px fixes it by making it perfectly round.

**Pull request merged**